docs/querying/sql.md

```diff
@@ -85,6 +87,121 @@ FROM clause, metadata tables are not considered datasources. They exist only in
 For more information about table, lookup, query, and join datasources, refer to the [Datasources](datasource.md)
 documentation.
 
+## PIVOT
+
+:::info
+The PIVOT operator is an [experimental feature](../development/experimental-features.md).
+:::
+
+The PIVOT operator carries out an aggregation and transforms rows into columns in the output.
+
+The following is the general syntax for the PIVOT operator. Note that the PIVOT operator is enclosed in parentheses and forms part of the FORM clause of the query.
+
+```sql
+PIVOT (aggregation_function(column_to_aggregate)
+  FOR [column_with_values_to_pivot]
+  IN ( [pivoted_column1], [pivoted_column2] ...)
+)
+```
+
+PIVOT syntax parameters:
+
+* `aggregation_function`: An aggregation function such as SUM, COUNT, MIN, MAX, or AVG.
```

Comment:
```suggestion
* `aggregation_function`: An aggregation function, such as SUM, COUNT, MIN, MAX, or AVG.
```

Reply:
Also the Google style guide says that the first letter after a colon isn't capitalized

Reply:
In the Druid docs, most of description lists with bulleted run-in heading start text that follows colons with a capital letter. I can propose an exception to the style guide.